src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -100,10 +100,15 @@ const StyledColumn = styled(Column)`
   position: relative;
 `;
 
+// Don't show the send email checkbox for statuses ingepland, hreopend, afgehandeld and geannuleerd
+export const EMAIL_CHECKBOX_EXCLUDED_STATES = ['ingepland', 'reopened', 'o', 'a'];
+const canSendMail = status => !EMAIL_CHECKBOX_EXCLUDED_STATES.includes(status);
```

Comment:
Looking at the statuses and the ticket https://datapunt.atlassian.net/browse/SIG-2929, there seems to be a discrepancy. The ticket mentions that the checkbox should only be shown when the current status equals Gemeld, In afwachting van behandeling, In behandeling, On hold, Extern verzoek tot afhandelen. Considering the list of statuses, this means that the checkbox should be shown when the status code equals either 'm', 'i', 'b' or 'closure requested' and should not be shown when the status code equals either 'o', 's', 'ingepland', 'a', 'reopen requested', 'reopened', 'ready to send', 'sent', 'send failed' or 'done external'.

The logic in this PR, whoever, differs from that. Can you explain why it is different?

Reply:
I used the last message from our PO: `belangrijkste is het NIET tonen van het vinkje bij afgehandeld, ingepland en heropend`. I added though the status `a` to it because for this status the message says that no mail will be sent.